admin/bin/qserv-data-loader.py

```diff
@@ -257,59 +246,17 @@ def run(self):
         return 0
 
 
-    def _czarDbConnect(self):
-        """
-        Connect to czar mysql server, returns connection or throws exception.
-        """
-        kws = dict(local_infile=1)
-        if self.args.mysqlHost:
-            kws['host'] = self.args.mysqlHost
-        if self.args.user:
-            kws['user'] = self.args.user
-        if self.args.password:
-            kws['passwd'] = self.args.password
-        if self.args.mysqlPort:
-            kws['port'] = self.args.mysqlPort
-        if self.args.mysqlSocket:
-            kws['unix_socket'] = self.args.mysqlSocket
-
-        try:
-            return mysql.Connection(**kws)
-        except Exception as exc:
-            logging.critical('Failed to connect to mysql database: %s', exc)
-            raise
-
-
-    def _workerDbConnect(self, nodeName, qservAdmin):
+    def _wmgrConnect(self, nodeName, qservAdmin):
         """
         Connect to worker mysql server, returns connection or throws exception.
         """
-
         if qservAdmin:
             # will get node info from CSS
-            wAdmin = NodeAdmin(name=nodeName, qservAdmin=qservAdmin)
+            nodeAdmin = NodeAdmin(name=nodeName, qservAdmin=qservAdmin, wmgrSecretFile=self.args.secret)
         else:
-            # no CSS, use node name as host name for direct mysql connection
-            mysqlConn = str(self.args.mysqlPort)
-            wAdmin = NodeAdmin(host=nodeName, mysqlConn=mysqlConn)
-
-        kws = dict(local_infile=1)
-        if self.args.user:
-            kws['user'] = self.args.user
-        if self.args.password:
-            kws['passwd'] = self.args.password
-
-        try:
-            # this will return db.Db instance
-            db = wAdmin.mysqlConn(**kws)
-            self._dbs.append(db)
-            db.connect()
-            # extract mysql connection object from Db, this is a hack to be used for now
-            # in the future we'll need to use Db API directly in data loader
-            return db._conn
-        except Exception as exc:
-            logging.critical('Failed to connect to mysql database: %s', exc)
-            raise
+            # no CSS, use node name as host name
```

Comment:
Mostly for testing. Again, currently data loader does not require CSS to be there, you can load data without CSS just fine. I hope that does not complicate things too much.